virttest/virsh.py

```diff
@@ -3099,6 +3099,18 @@ def cpu_compare(xml_file, **dargs):
     return command("cpu-compare %s" % xml_file, **dargs)
 
 
+def hypervisor_cpu_compare(xml_file, options="", **dargs):
```

Comment:
request to have param for `virttype`, `emulator`, `arch` and `machine` and process it here instead in testcase. `options` would be used for `--error`